### ISSUE TITLE
Exclude deleted projects from cron jobs

### ISSUE THREAD

**Issue by mmcev106** [vanderbilt/redcap-external-modules#207]
Would the best solution be to exclude deleted projects from `getProjectsWithModuleEnabled()`?  If so, should we update the docs to stress using this method inside crons?  Should we also exclude any other project statuses, perhaps "archived"?

**Comment by scottjpearson** [comment]
Should the external-module settings be deleted when a project is deleted, too? I think so. I'm playing ignorance about the archived projects, though. I don't use that feature much.

**Comment by mmcev106** [comment]
Yes, we likely should.  I just created the following issue to remember that:

https://github.com/vanderbilt/redcap-external-modules/issues/208

**Comment by taylorr4** [comment]
@mmcev106 you might want to add a new join in `getProjectsWithModuleEnabled()` so that you join with redcap_projects and add "and date_deleted is null" to the WHERE clause. That would easily exclude projects that have been "deleted" (i.e., are in the "recycle bin" for 30 days prior to being removed from the database fully).

**Comment by mmcev106** [comment]
After speaking to @taylorr4 I believe we should disable crons on archived projects as well.  @taylorr4 suggested only running crons for statuses 0 and 1.
